README.md

```diff
@@ -17,82 +17,100 @@ This version of the library supports the following functionality:
-| *onAuthenticated* | Function | Optional | Callback called when the operation is completed or an error occurs. |
-| *route* | String | Optional | The login endpoint's route. If not specified or `null`, the default route `"/"` is used. |
-| *testDevice* | Boolean | Optional | `True` if it is a test device. `False` by default. For more information, please see [here](https://developer.amazon.com/docs/dash/test-device-purchases.html) and the [Test Orders section](#test-orders). |
+#### Parameters ####
 
-The method returns nothing. A result of the operation may be obtained via the [onAuthenticated](#callback-onauthenticatederror-response) callback if specified in this method.
+| Parameter | Type | Required | Description |
+| --- | --- | --- | --- |
+| *rocky* | [Rocky](https://github.com/electricimp/Rocky) instance | Yes | An instance of the [Rocky](https://github.com/electricimp/Rocky) library |
+| *deviceModel* | String | Yes | For information, please see [the Amazon documentation](https://developer.amazon.com/docs/dash/lwa-web-api.html#integrate-with-the-lwa-sdk-for-javascript) |
+| *deviceSerial* | String | Yes | For information, please see [the Amazon documentation](https://developer.amazon.com/docs/dash/lwa-web-api.html#integrate-with-the-lwa-sdk-for-javascript) |
+| *onAuthenticated* | Function | Optional | Callback called when the operation is completed or an error occurs. See below |
+| *route* | String | Optional | The login endpoint's route. Default: `"/"` |
+| *testDevice* | Boolean | Optional | `true` if it is a test device; `false` by default. For more information, please see [the Amazon documentation](https://developer.amazon.com/docs/dash/test-device-purchases.html) and the [Test Orders](#test-orders) section |
 
-#### Callback: onAuthenticated(*error, response*) ####
+#### onAuthenticated Callback Parameters ####
 
-| Parameter | Data Type | Description |
+| Parameter | Type | Description |
 | --- | --- | --- |
-| *error* | Integer | `0` if the authentication is successful, an [error code](#error-code) otherwise. See possible HTTP error codes [here](https://developer.amazon.com/docs/login-with-amazon/authorization-code-grant.html#access-token-errors). |
-| *response* | Table | Key-value table with the response provided by Amazon server. May be `null`. [See here about the response format](https://developer.amazon.com/docs/login-with-amazon/authorization-code-grant.html#access-token-response). Also may contain error details described [here](https://developer.amazon.com/docs/login-with-amazon/authorization-code-grant.html#access-token-errors) and [here](https://developer.amazon.com/docs/login-with-amazon/authorization-code-grant.html#authorization-errors). |
+| *error* | Integer | `0` if the authentication is successful, otherwise an [error code](#error-code) otherwise. Possible HTTP error codes [are listed here](https://developer.amazon.com/docs/login-with-amazon/authorization-code-grant.html#access-token-errors) |
```

Comment:
`otherwise` written twice. I suppose it is a mistake

Reply:
Fixed